### ISSUE TITLE
My domain is being blocked by your block list

### ISSUE THREAD

**Issue by jmchaves** [uBlockOrigin/uAssets#14531]
### Prerequisites

- [X] This is not a [support issue or a question](https://old.reddit.com/r/uBlockOrigin/)
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uAssets/issues) to avoid opening a duplicate issue
- [X] I am running the latest version of uBlock Origin

### I tried to reproduce the issue when...

- [X] uBO is the only extension
- [X] uBO with default lists/settings
- [X] using a new, unmodified browser profile

### URL(s) where the issue occurs

`https://www.springbot.com`

### Describe the issue

For some reason my domain is in the blocklist and the page is not loading correctly. We do not run or display ads in our site.

### Screenshot(s)

![Screenshot from 2022-08-25 17-09-28](https://user-images.githubusercontent.com/2567814/186769434-f130bfa2-74f8-4bc2-affd-d1b255a2f036.png)

![Screenshot from 2022-08-25 17-08-29](https://user-images.githubusercontent.com/2567814/186769458-4f796251-3fcf-4a95-9ce6-808d1a113b47.png)

### uBlock Origin version

uBlock Origin 1.44.0

### Browser name and version

Firefox 104.0 and Chrome 104.0.5112.101

### Settings

I haven't made any changes.

### Notes

We just need to remove the domain from blocklist and our users will be able to load the page.

**Comment by troysjanda** [comment]
Looks like its due to Peter Lowe's  list, Might try the maintainers list support. https://pgl.yoyo.org/adservers/#submit

**Comment by SimplesmenteMaria** [comment]
@troysjanda uBlock Origin overwrites problematic filters from the lists that are enabled by default (like Peter's). 

Only third-party requests to OP's domain should be blocked, not his whole site.

**Comment by MasterKia** [comment]
> For some reason
> We do not run or display ads in our site.

Your site is "Email marketing and paid advertising platform".

If the block gets reduces to third-party then this site can use click tracker links (redirect links) in emails and track users.

**Comment by troysjanda** [comment]
That's what I said in my deleted post. But someone else said differently
and I removed my post.

**Comment by Yuki2718** [comment]
@pgl

**Comment by pgl** [comment]
@Yuki2718 Thanks for the ping. And thanks to @MasterKia, my response would have been be almost identical to yours.

**Comment by jmchaves** [comment]
@MasterKia , that is the purpose of our site. Users manage their ads and email campaigns through our web application. In this case, makes no sense block the site.

**Comment by okiehsch** [comment]
His point was
`If the block gets reduced to third-party then this site can use click tracker links (redirect links) in emails and track users.`

> Users manage their ads and email campaigns

Your users, who manage ads and email campaigns,  use an adblocker, well...


> ....the only people who would complain genuinely are clients of these sites. They are free to disable their blockers to make these sites work properly.
https://github.com/uBlockOrigin/uAssets/pull/10149#issuecomment-951821141

**Comment by troysjanda** [comment]
@okiehsch I viewed your site source and there are scripts for Facebook pixel and adsense tracking so even if there are not ads displayed your clearly tracking users with a few methods.

**Comment by okiehsch** [comment]
My site's source?

**Comment by MasterKia** [comment]
I think he meant to tag the OP.

**Comment by troysjanda** [comment]
> 

yes right clicking on your site page and view source. its a feature of Firefox and every browser. I count at lease 4 FB pixel tracking scripts and 2 adsense. Also in Ubo Logger there are Filters in place by Easylist, eaasy Privacy, Peter lowe's list to name a few.

**Comment by troysjanda** [comment]
Sorry yes I meant the OP please forgive me

**Comment by okiehsch** [comment]
No problem.

**Comment by jmchaves** [comment]
@MasterKia , I understand the domain is blocked in others sites, but why our actually site?

**Comment by troysjanda** [comment]
Read the comments.

**Comment by MasterKia** [comment]
@jmchaves Could you test the fix @okiehsch suggested?
https://github.com/gorhill/uBlock/wiki/How-to-mark-a-web-site-as-trusted#click-the-big-power-button

If it works, then suggest it to your customer too.

**Comment by jmchaves** [comment]
@MasterKia , that works for our current customers, but our startup company hopes to get more potential customers and blocking our site sends them away. Again, the purpose of our web application is to manage their ads and campaigns. Similar to `https://mailchimp.com/`, which btw is not blocked by uBlock.

**Comment by troysjanda** [comment]
@jmchaves I think that you are missing the issue, As I stated in my above comment 

>I viewed your site source and there are scripts for Facebook pixel and adsense tracking so even if there are not ads displayed your clearly tracking users with a few methods.

And

>I count at lease 4 FB pixel tracking scripts and 2 adsense. Also in Ubo Logger there are Filters in place by Easylist, Easy Privacy, and Peter lowe's list to name a few.

there are tracking elements on your page.

As okiehsch also said in his comment.

>His point was
If the block gets reduced to third-party then this site can use click tracker links (redirect links) in emails and track users.

This is why it was declined here as well at by pgl himself.
